docs/cli.md

```diff
@@ -204,3 +204,35 @@ Prevent errors when pattern is unmatched.
 ## `--no-plugin-search`
 
 Disable plugin autoloading.
+
+## `--cache`
+
+If this option is enabled, the following values are used as cache keys and the file is formatted only if one of them is changed.
+
+- Prettier version
+- Options
+- Node.js version
+- (if `--cache-strategy` is `metadata`) file metadata, such as timestamps
+- (if `--cache-strategy` is `content`) content of the file
+
+```bash
+prettier --write --cache src
+```
+
+Running Prettier without `--cache` will delete the cache.
+
+Also, since the cache file is stored in `./node_modules/.cache/prettier/.prettier-cache`, so you can use `rm -f . /node_modules/.cache/prettier/.prettier-cache` to remove them manually.
```

Comment:
```suggestion
Also, since the cache file is stored in `./node_modules/.cache/prettier/.prettier-cache`, so you can use `rm ./node_modules/.cache/prettier/.prettier-cache` to remove it manually.
```

- Removed accidental space.
- Removed `-f` so one gets an error message if the file is not found (for example you’re in a subdirectory and need `../`).
- Changed “them” to “it”.